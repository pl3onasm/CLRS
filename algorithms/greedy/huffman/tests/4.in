aaaaaaaaa
bbbbbbbbb
ccccccccc
ddddddddd
eeeeeeeee
fffffffff
ggggggggg
hhhhhhhhh
iiiiiiiii
jjjjjjjjj
kkkkkkkkk
lllllllll
mmmmmmmmm
nnnnnnnnn
ooooooooo
ppppppppp
qqqqqqqqq
rrrrrrrrr
sssssssss
ttttttttt
uuuuuuuuu
vvvvvvvvv
wwwwwwwww
xxxxxxxxx
yyyyyyyyy
zzzzzzzzz