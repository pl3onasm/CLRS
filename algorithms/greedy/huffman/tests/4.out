CODEWORD TABLE

char      freq       code
   u         9       0000
   q         9       0001
   r         9       0010
   s         9       0011
   f         9       0100
   l         9       0101
   o         9       01100
   m         9       01101
   k         9       01110
   c         9       01111
   a         9       10000
   z         9       10001
   g         9       10010
   n         9       10011
   t         9       10100
   i         9       10101
   x         9       10110
   d         9       10111
   e         9       11000
   j         9       11001
   y         9       11010
   b         9       11011
   w         9       11100
   h         9       11101
   v         9       11110
   p         9       11111


HUFFMAN TREE

-0-0-0-0 [9] 'u'
-0-0-0 [18]
-0-0-0-1 [9] 'q'
-0-0 [36]
-0-0-1-0 [9] 'r'
-0-0-1 [18]
-0-0-1-1 [9] 's'
-0 [90]
-0-1-0-0 [9] 'f'
-0-1-0 [18]
-0-1-0-1 [9] 'l'
-0-1 [54]
-0-1-1-0-0 [9] 'o'
-0-1-1-0 [18]
-0-1-1-0-1 [9] 'm'
-0-1-1 [36]
-0-1-1-1-0 [9] 'k'
-0-1-1-1 [18]
-0-1-1-1-1 [9] 'c'
root [234]
-1-0-0-0-0 [9] 'a'
-1-0-0-0 [18]
-1-0-0-0-1 [9] 'z'
-1-0-0 [36]
-1-0-0-1-0 [9] 'g'
-1-0-0-1 [18]
-1-0-0-1-1 [9] 'n'
-1-0 [72]
-1-0-1-0-0 [9] 't'
-1-0-1-0 [18]
-1-0-1-0-1 [9] 'i'
-1-0-1 [36]
-1-0-1-1-0 [9] 'x'
-1-0-1-1 [18]
-1-0-1-1-1 [9] 'd'
-1 [144]
-1-1-0-0-0 [9] 'e'
-1-1-0-0 [18]
-1-1-0-0-1 [9] 'j'
-1-1-0 [36]
-1-1-0-1-0 [9] 'y'
-1-1-0-1 [18]
-1-1-0-1-1 [9] 'b'
-1-1 [72]
-1-1-1-0-0 [9] 'w'
-1-1-1-0 [18]
-1-1-1-0-1 [9] 'h'
-1-1-1 [36]
-1-1-1-1-0 [9] 'v'
-1-1-1-1 [18]
-1-1-1-1-1 [9] 'p'

